build.sbt

```diff
@@ -43,7 +43,7 @@ lazy val api = crossProject(JSPlatform, JVMPlatform, NativePlatform)
     name := "cldr-api",
     scalaVersion := scalaVersion212,
     description := "scala-java-locales cldrl api",
-    crossScalaVersions := Seq(scalaVersion212, "2.13.14", "3.4.1"),
+    crossScalaVersions := Seq(scalaVersion212, "2.13.14", "3.3.3"),
```

Comment:
Otherwise we would force all users of this library off Scala LTS, which is probably not desirable.

Reply:
👍